packages/server/src/unstable-core-do-not-import/stream/sse.ts

```diff
@@ -346,7 +346,7 @@ export function sseStreamConsumer<TConfig extends ConsumerConfig>(
           _es = null;
         });
         eventSource.addEventListener('error', (event) => {
-          if (eventSource.readyState === EventSource.CLOSED) {
+          if (eventSource.readyState === eventSource.CLOSED) {
```

Comment:
```suggestion
          if (eventSource.readyState === opts.EventSource.CLOSED) {
```

Reply:
Can't make this change, results in a type error. The types have `CLOSED` existing on the instance only. https://github.com/trpc/trpc/blob/9f1e419518a121a77c2437a986d75aea265e467a/packages/server/src/unstable-core-do-not-import/stream/sse.types.ts#L24-L27

Reply:
Test it with `@canary`, I think this should work